src/main/java/beans/menus/EntityListResponse.java

```diff
@@ -187,7 +190,15 @@ private EntityBean processEntity(TreeReference entity, EntityScreen screen, Eval
         for (DetailField field : fields) {
             Object o;
             o = field.getTemplate().evaluate(context);
-            data[i] = o;
+            if(o instanceof GraphData) {
+                try {
+                    data[i] = GraphUtil.getHTML((GraphData) o, "");
+                } catch (GraphException e) {
+                    data[i] = "Error loading graph " + e;
```

Comment:
if we're going to stick with this, this should at the least return valid html
`<html><body>Error loading...</body></html>`